Codeinwp/neve
# fix: duplicated IDs for legacy search last menu item

### Summary
Fixes issue with duplicated ID for legacy last menu item search.


### Will affect visual aspect of the product

NO

### Test instructions


- Import a demo with legacy last menu items (e.g. neve-shop)  
- Make sure you display primary menu both in the mobile sidebar menu and in the desktop header rows
- Front end should not have duplicated id `nv-menu-item-search`. 


Closes #1656.
